Reddit Post Templates/mainTemplate.md

```diff
@@ -20,6 +20,6 @@ Tell us your thoughts!
 
 {prevPostSect}
 
-^(* Pick and Win rates are obtained from [here](https://starcraft2coop.com/resources/stats) at the time of this post)
+^(* Pick and Win rates are obtained from) [^here](https://starcraft2coop.com/resources/stats) ^(at the time of this post)
```

Comment:
This unfortunately breaks the format for the new reddit layout. Has to be `^[]` instead I think. I'll merge this if you can change that in this PR.

Reply:
Also kinda messes up the end brackets in new layout

Reply:
Sorry about that! It looks like either that or adding parentheses around the bracketed "here" work, I think. If you let me know which you'd prefer I'll do that. Also, do you want me to revert the changes to config.yml in my pull request? Not sure if it would make merging easier.

Thanks for letting me practice my git and github skills!

Reply:
This is what the parentheses solution would look like, similar to the other link.

```suggestion
^(* Pick and Win rates are obtained from) [^(here)](https://starcraft2coop.com/resources/stats) ^(at the time of this post)
```